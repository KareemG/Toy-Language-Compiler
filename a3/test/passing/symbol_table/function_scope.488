{
    var ret : integer
    
    function foo : integer
    {
        var a : integer
        a := 1

        {
            var b : integer

            {
                return with 2
            }

            return with a + b
        }
    }

    ret := foo
}