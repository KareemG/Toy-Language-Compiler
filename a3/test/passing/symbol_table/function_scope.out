{
 Symbols:
 ret: integer
 foo: integer
 foo : integer{
 Symbols:
 a: integer
 {
 Symbols:
 b: integer
 {
 Symbols:
 }
 }
 }
